test/bdd/features/vc_e2e_api.feature

```diff
@@ -21,12 +21,15 @@ Feature: Using VC REST API
+      | myprofile_ud         | university_degree.json       |                                                                  |                                                                                          | ProofValue      |       {"driverURL":"http://uni-registrar-web:9080/1.0/register?driver-did-method-rest"}                                                                       | did:trustbloc   |   JsonWebSignature2020  |           |
+      | myprofile_prc        | permanent_resident_card.json |                                                                  |                                                                                          | ProofValue      |          {"driverURL":"http://uni-registrar-web:9080/1.0/register?driver-did-method-rest"}                                                                    | did:trustbloc   |   JsonWebSignature2020  |           |
+      | myprofilewithdidv1   | university_degree.json       |                                                                  |                                                                                          | JWS             |       {"driverURL":"http://uni-registrar-web:9080/1.0/register?driver-universalregistrar/driver-did-v1","options": {"ledger": "test", "keytype": "ed25519"}}  | did:v1:test:nym |    Ed25519Signature2018 |           |
+      | myprofilewithdidelem | university_degree.json       | did:elem:EiAWdU2yih6NA2IGnLsDhkErZ8aQX6b8yKt7jHMi-ttFdQ          | 5AcDTQT7Cdg1gBvz8PQpnH3xEbLCE1VQxAJV5NjVHvNjsZSfn4NaLZ77mapoi4QwZeBhcAA7MQzaFYkzJLfGjNnR | JWS             |                                                                                                                                                               | did:elem        |    Ed25519Signature2018 |           |
+      | myprofilewithdidsov  | university_degree.json       |                                                                  |                                                                                          | JWS             |       {"driverURL":"https://uniregistrar.io/1.0/register?driver-universalregistrar/driver-did-sov","options": {"network":"danube"}}                           | did:sov:danube  |    Ed25519Signature2018 |           |
+      | myprofile_ed25519    | university_degree.json       |                                                                  |                                                                                          | JWS             |                                                                                                                                                               | did:trustbloc   |   JsonWebSignature2020  |  Ed25519  |
```

Comment:
can we use current profiles [myprofile_ud,myprofile_prc ]

Reply:
sure.

Reply:
myprofile_ud for Ed25519   and myprofile_prc  for P256

Reply:
we also need a prc profile with ed25519. need to show both. @rolsonquadras @fqutishat

Reply:
[Added](https://github.com/trustbloc/edge-service/compare/2930edf978b9485a9e844730946e04a38e4817ce..4cd60a8250724ba9440a3a9bc740599c492d61e4#diff-d0c4c7ca6e3fe2063f6b701d16489b4bR30-R34) profiles for UD/PRC with ed25519/P256.